grails-forge-core/src/main/java/org/grails/forge/feature/view/Scaffolding.java

```diff
@@ -57,12 +57,12 @@ public String getDescription() {
 
     @Override
     public String getDocumentation() {
-        return "https://grails.github.io/scaffolding/latest/groovydoc/";
+        return "https://github.com/grails/scaffolding";
     }
 
     @Override
     public String getThirdPartyDocumentation() {
-        return "https://grails-fields-plugin.github.io/grails-fields/latest/guide/index.html";
+        return "https://github.com/gpc/fields";
```

Comment:
Should be https://gpc.github.io/fields/, but the documentation publishing is currently not working.

Reply:
I've kept the GitHub README.md link for fields